pkg/printers/internalversion/printers_test.go

```diff
@@ -6206,6 +6208,322 @@ func TestPrintFlowSchema(t *testing.T) {
+		name          string
+		resourceClaim resourceapis.ResourceClaim
+		expected      []metav1.TableRow
+	}{
+		{
+			name: "ResourceClaim with Pending State",
+			resourceClaim: resourceapis.ResourceClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "resourceclaim",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: resourceapis.ResourceClaimSpec{
+					Devices: resourceapis.DeviceClaim{
+						Requests: []resourceapis.DeviceRequest{
+							{
+								Name:            "deviceRequest",
+								DeviceClassName: "deviceClass",
+								AllocationMode:  resourceapis.DeviceAllocationModeExactCount,
+								Count:           1,
+							},
+						},
+					},
+				},
+			},
+			// Columns: Name, State, Age
+			expected: []metav1.TableRow{{Cells: []interface{}{"resourceclaim", "pending", "0s"}}},
+		},
+		{
+			name: "ResourceClaim with Allocated State and Deallocation State",
+			resourceClaim: resourceapis.ResourceClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "resourceclaim",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: resourceapis.ResourceClaimSpec{
+					Devices: resourceapis.DeviceClaim{
+						Requests: []resourceapis.DeviceRequest{
+							{
+								Name:            "deviceRequest",
+								DeviceClassName: "deviceClass",
+								AllocationMode:  resourceapis.DeviceAllocationModeExactCount,
+								Count:           1,
+							},
+						},
+					},
+				},
+				Status: resourceapis.ResourceClaimStatus{
+					Allocation: &resourceapis.AllocationResult{
+						Controller: "dra.example.com",
+					},
+					DeallocationRequested: true,
+				},
+			},
+			// Columns: Name, State, Age
+			expected: []metav1.TableRow{{Cells: []interface{}{"resourceclaim", "allocated,deallocating", "0s"}}},
+		},
+		{
+			name: "ResourceClaim with Allocated and Reserved State",
+			resourceClaim: resourceapis.ResourceClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "resourceclaim",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
+				},
+				Spec: resourceapis.ResourceClaimSpec{
+					Devices: resourceapis.DeviceClaim{
+						Requests: []resourceapis.DeviceRequest{
+							{
+								Name:            "deviceRequest",
+								DeviceClassName: "deviceClass",
+								AllocationMode:  resourceapis.DeviceAllocationModeExactCount,
+								Count:           1,
+							},
+						},
+					},
+				},
+				Status: resourceapis.ResourceClaimStatus{
+					Allocation: &resourceapis.AllocationResult{
+						Controller: "dra.example.com",
+					},
+					ReservedFor: []resourceapis.ResourceClaimConsumerReference{
+						{
+							Resource: "pods",
+							Name:     "pod-test",
+							UID:      types.UID("pod-test"),
+						},
+					},
+					DeallocationRequested: true,
+				},
+			},
+			// Columns: Name, State, Age
+			expected: []metav1.TableRow{{Cells: []interface{}{"resourceclaim", "allocated,reserved", "0s"}}},
```

Comment:
Is it possible to manage human duration to put other than `0s`?

Reply:
I will change the result of all Age fields to 5m, which may be easier to understand.